packages/javascript/plotlywidget/package.json

```diff
@@ -35,7 +35,7 @@
     "typescript": "~3.1.1"
   },
   "dependencies": {
-    "plotly.js": "^1.51.2",
+    "plotly.js": "https://64237-45646037-gh.circle-artifacts.com/0/plotly.js.tgz",
```

Comment:
was this changed when running the `setup.py` or did you change it manually? I'm asking because I had first tried to change this line manually by giving a URL as here, but then when I ran `npm install` npm was throwing an error.

Reply:
`setup.py` did this.